cli/src/main.rs

```diff
@@ -0,0 +1,95 @@
+use netsim_embed::{run, Ipv4Range, NetworkBuilder, Wire};
+use netsim_embed_cli::{Command, Event};
+use std::path::PathBuf;
+use std::time::Duration;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub struct Opts {
+    #[structopt(long)]
+    topology: String,
+    #[structopt(long)]
+    client: Option<PathBuf>,
+    #[structopt(long)]
+    server: PathBuf,
+    #[structopt(long)]
+    delay_ms: Option<u64>,
+}
+
+fn main() {
```

Comment:
So the idea is to build the examples, and then use the cli to run them? That's not exactly obvious and might warrant a few examples/comments.

Also: Having a dedicated example without the `cli` in the middle might also be nice. I can add that.

Reply:
Hmm, maybe I was wrong. How's that supposed to work?
```
✗ cargo run -- --client ../target/debug/examples/tcp_client --server ../target/debug/examples/tcp_server --topology m1
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `/home/ow/src/netsim-embed/target/debug/netsim-embed-cli --client ../target/debug/examples/tcp_client --server ../target/debug/examples/tcp_server --topology m1`
```

Reply:
Well the examples are more like tests and this was mainly to reduce boilerplate as using netsim requires some boilerplate.